Makefile

```diff
@@ -49,6 +50,18 @@ DEPLOY_TAG := $(or $(DEPLOY_TAG), "")
 IMAGE_NAME=test-infra
 IMAGE_REG_NAME=quay.io/itsoiref/$(IMAGE_NAME)
 
+# oc deploy
+OC_MODE := $(or $(OC_MODE),"")
```

Comment:
i think u can set this flag if u get oc_user, what do u think?

Reply:
thought that but preferred to be more explicit, maybe we'll decide to change user field to token